server/app/views/AlertComponent.java

```diff
@@ -0,0 +1,123 @@
+package views;
+
+import static j2html.TagCreator.div;
+import static j2html.TagCreator.h2;
+import static j2html.TagCreator.h3;
+import static j2html.TagCreator.h4;
+import static j2html.TagCreator.h5;
+import static j2html.TagCreator.h6;
+import static j2html.TagCreator.p;
+
+import com.google.common.collect.Lists;
+import j2html.tags.specialized.DivTag;
+import java.util.Optional;
+import services.AlertType;
+import views.style.BaseStyles;
+
+/**
+ * Render a USWDS Alert Component
+ *
+ * @see <a
+ *     href="https://designsystem.digital.gov/components/alert">https://designsystem.digital.gov/components/alert</a>
+ */
+public final class AlertComponent {
+  /**
+   * Heading levels that can be used for Alerts. H1 tags are limited to a single instance per page
+   * and and not allowed as headings.
+   */
+  public enum HeadingLevel {
+    H2,
+    H3,
+    H4,
+    H5,
+    H6
+  }
+
+  /**
+   * Makes a USWDS Alert component with the given text and optional title. Alert variant is
+   * determined by the {@link AlertType} passed in.
+   *
+   * @param alertType The type of {@link AlertType} alert to show
+   * @param text The text to include in the alert.
+   * @param title An optional title to be included in the alert.
+   * @param hidden Whether or not to set the hidden property on the component.
+   * @param headingLevel Allow configuring the heading level between H2-H6. Default is H4
+   * @param classes One or more additional classes to apply to the USWDS Alert component.
+   * @return DivTag containing the alert.
+   */
+  public static DivTag renderFullView(
```

Comment:
Nitpick: these method names pretty un-intuitive to me... I liked "makeAlert", or "renderAlert". I guess it partly depends on the answer to my question below about the slim option, but either way "view" sounds like something different from an alert to me.

Reply:
Now that I moved this to a class by itself, having alert in the method name seems redundant.

Reply:
I changed them to `renderFullAlert` and `renderSlimAlert`. Let me know if that works.

Reply:
👍  Thanks!